features/steps/test_store_resource.py

```diff
@@ -0,0 +1,248 @@
+                        detail="Hear thus part probably that.",
+                        agent=Agent(address="kurt16@example.org", role="ocr processing"),
+                    ),
+                ],
+                metadata_files=[
+                    FileMetadata(
+                        id="_0193972b-e4a4-7985-abe2-f3f1259b78ec",
+                        use="TECHNICAL",
+                        ref=FileReference(
+                            locref="../metadata/00000001.source.jpg.mix.xml",
+                            mdtype="NISOIMG",
+                        ),
+                    ),
+                    FileMetadata(
+                        id="_0193972b-e4ae-73eb-848d-5f8893b68253",
+                        use="TECHNICAL",
+                        ref=FileReference(
+                            locref="../metadata/00000001.access.jpg.mix.xml",
+                            mdtype="NISOIMG",
+                        ),
+                    ),
+                    FileMetadata(
+                        id="_0193972b-e572-7107-b69c-e2f4c660a9aa",
+                        use="TECHNICAL",
+                        ref=FileReference(
+                            locref="../metadata/00000001.plaintext.txt.textmd.xml",
+                            mdtype="TEXTMD",
+                        ),
+                    ),
+                ],
+                data_files=[
+                    FileMetadata(
+                        id="_be653ff450ae7f3520312a53e56c00bc",
+                        mdid="_0193972b-e4a4-7985-abe2-f3f1259b78ec",
+                        use="SOURCE",
+                        ref=FileReference(
+                            locref="../data/00000001.source.jpg",
+                            mimetype="image/jpeg",
+                        ),
+                    ),
+                    FileMetadata(
+                        id="_7e923d9c33b3859e1327fa53a8e609a1",
+                        groupid="_be653ff450ae7f3520312a53e56c00bc",
+                        mdid="_0193972b-e4ae-73eb-848d-5f8893b68253",
+                        use="ACCESS",
+                        ref=FileReference(
+                            locref="../data/00000001.access.jpg",
+                            mimetype="image/jpeg",
+                        ),
+                    ),
+                    FileMetadata(
+                        id="_764ba9761fbc6cbf0462d28d19356148",
+                        groupid="_be653ff450ae7f3520312a53e56c00bc",
+                        mdid="_0193972b-e572-7107-b69c-e2f4c660a9aa",
+                        use="SOURCE",
+                        ref=FileReference(
+                            locref="../data/00000001.plaintext.txt",
+                            mimetype="text/plain",
+                        ),
+                    ),
+                ],
+            )
+        ]
+
+
+@dataclass
+class Context:
+    uow: UnitOfWork
+    translocator: Translocator
+    message_bus: MemoryMessageBus
```

Comment:
This seems to be missing `stored_event`. Also, all these are `None` at some point, so I'd include that in the type signatures. `uow`, `translocator`, and `message_bus` technically don't have to be see since you could instantiate the `Context` at the end of the `given` with those three and they'll never be `None`.